### PR TITLE
Adding missing [CanEditMultipleObject] attributes

### PR DESCRIPTION
**Pull request by ms-RistoRK**
This is for https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/issues/573

### CODE REVIEW

**Review by AMollis — CHANGES_REQUESTED** [review]
Riso thank your for changes.   A couple of requests

1. Update the "patch" number for all packages.  Scott has already branched for the "release". See https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/tree/releases/2024-03-20
2. Update changelog.md for all packages.  For each changelog.md you'll need to add a new "-development" header for the new patch version.  Keep the old headers but remove the "-development" tag from them.

**Comment by ms-RistoRK** [comment]
> Riso thank your for changes. A couple of requests
> 
> 1. Update the "patch" number for all packages, update the package.json.  Scott has already branched for the "release". See https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/tree/releases/2024-03-20
> 2. Update changelog.md for all packages.  For each changelog.md you'll need to add a new "-development" header for the new patch version.  Keep the old headers but remove the "-development" tag from them.

Done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/698/commits/01b806b8de183964d2833cc6147e3da5b0604a4d

File: org.mixedrealitytoolkit.core/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
+## [3.2.1-development] - 2024-04-11
+
+### Fixed
+
+* Fixed missing [CanEditMultipleObject] attributes as per Bug 573 [PR #698](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/698)
+
 ## [3.2.0-development] - 2024-03-20
```
Comment (AMollis) [inline org.mixedrealitytoolkit.core/CHANGELOG.md:11]: Only the top version should have -development....

Please remove -development for older releases.  

(Same comment for all other MD changelogs)

**Review by AMollis — CHANGES_REQUESTED** [review]

File: org.mixedrealitytoolkit.core/CHANGELOG.md (hunk 1 above)
Comment (ms-RistoRK) [reply org.mixedrealitytoolkit.core/CHANGELOG.md]: Done, please re-review whenever you have time.  Resolving this.

File: org.mixedrealitytoolkit.data/CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,13 @@
...
-## [1.0.3-development] - 2024-04-17
+## [1.0.4-development] - 2024-04-18
+
+### Fixed
+
+* Fixed missing [CanEditMultipleObject] attributes as per Bug 573 [PR #698](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/698)
+
+## [1.0.3] - 2024-04-17
 
 ### Fixed
```
Comment (AMollis) [inline org.mixedrealitytoolkit.data/CHANGELOG.md:13]: Please double check all your version numbers.  For example org.mixedrealitytoolkit.data v1.0.3 is not released yet, so 1.0.4 is not needed.

Please referred to the last release branch here:

https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/tree/releases/2024-03-20

**Review by AMollis — CHANGES_REQUESTED** [review]

File: org.mixedrealitytoolkit.data/CHANGELOG.md (hunk 2 above)
Comment (ms-RistoRK) [reply org.mixedrealitytoolkit.data/CHANGELOG.md]: Done, I think, Can you please re-review? resolving this in the meantime.

**Review by AMollis — APPROVED** [review]

**Pull request merged**